p2panda-rs/src/secret_group/group.rs

```diff
@@ -189,52 +189,13 @@ impl SecretGroup {
     }
 
     /// Return the current group members.
-    pub fn members(&self) -> Result<Vec<Author>, SecretGroupError> {
-        let read_author = |member: Credential| {
-            let public_key = PublicKey::from_bytes(member.identity())
-                .map_err(|_| SecretGroupError::InvalidMemberPublicKey)?;
-
-            // It's safe to unwrap here since we just constructed the public key ourselves
-            Ok(Author::try_from(public_key).unwrap())
-        };
-
-        self.mls_group
-            .members()
-            .unwrap()
-            .into_iter()
-            .map(read_author)
-            .collect()
+    pub fn members(&self) -> Vec<&KeyPackage> {
+        self.mls_group.members()
```

Comment:
The new MLS spec removes members identified by a `KeyPackage` hash. To generate the hashes we need to return the `KeyPackage` of all members here instead of `Author`.

I'm sure there is a way to make this come closer to p2panda as soon as integrate it with our operations but on that level here we have to do it like that now.